aspnetcore/razor-components/components.md

```diff
@@ -320,7 +321,7 @@ protected override void OnInit()
 }
 ```
 
-`OnParametersSetAsync` and `OnParametersSet` are called when a component has received parameters from its parent and the values are assigned to properties. These methods are executed after `OnInit` during component initialization.
+`OnParametersSetAsync` and `OnParametersSet` are called when a component has received parameters from its parent and the values are assigned to properties. These methods are executed after `OnInit` during component initialization:
```

Comment:
These methods are executed after component initialization and then each time the component is then rendered: